### DISCUSSION TITLE
Need Help DNS/SSL

### DISCUSSION THREAD

**Discussion by benniledl** [processone/ejabberd#3870] [Questions] [answered]
Hello I am setting up a new ejabberd server

domain is: ejabberd.******

I have an ssl certificate for ejabberd.****** and 2 srv dns records

_xmpp-client._tcp.ejabberd.******. IN SRV 5 0 5222 ejabberd.******.
_xmpp-server._tcp.ejabberd.******. IN SRV 5 0 5269 ejabberd.******.

Now in my log there are these warnings:

ejabberd_pkix:check_domain_certfiles/1:312 No certificate found matching proxy.ejabberd.******
ejabberd_pkix:check_domain_certfiles/1:312 No certificate found matching upload.ejabberd.******
ejabberd_pkix:check_domain_certfiles/1:312 No certificate found matching conference.ejabberd.******
ejabberd_pkix:check_domain_certfiles/1:312 No certificate found matching pubsub.ejabberd.******

Can someone help me with:
What DNS Records do I need to make for those subdomains?
Do I even have to make DNS Records for these subdomains in order to create certificates for them? Yeah right?

I don't think you need that info but client will be conversejs running on Apache2 Port 443.

I haven't used ejabberd before so your help would be much appreciated.

Thank you,
Benni

**Comment by badlop** [comment, marked as answer]
> What DNS Records do I need to make for those subdomains?

This question is common to any XMPP server, so this documentation gives you some ideas:
https://prosody.im/doc/dns#subdomains

**Reply by benniledl** [reply]
Thank you very much :)
I didn't look into prosody's documentation. Maybe someone can add this to Ejabberds Docs?
